app/adapters/application.js

```diff
@@ -0,0 +1,4 @@
+import JSONAPIAdapter from 'ember-data/adapters/json-api';
+
+export default JSONAPIAdapter.extend({
```

Comment:
Since this is the default adapter, the application adapter file and its associated unit tests can be removed. (this is basically what would happen if the file didn't exist at all)